lona/html/parsing.py

```diff
@@ -107,24 +108,46 @@ def get_node_class(self, tag_name, attributes):
         # generic nodes
         return Node
 
-    def handle_starttag(self, tag, attrs):
-        node_kwargs = {}
+    def parse_attribute_list(self, attribute: str) -> List[str]:
+        attribute_list = []
 
-        # node attributes
-        for name, value in attrs:
-            if value is None:
-                value = 'true'
+        for i in attribute.split(' '):
+            i = i.strip()
 
-            node_kwargs[name] = value
+            if not i:
+                continue
 
+            attribute_list.append(i)
+
+        return attribute_list
+
+    def handle_starttag(self, tag, attrs):
         # tag overrides
+        node_kwargs = {}
+
         node_kwargs['tag_name'] = tag
         node_kwargs['self_closing_tag'] = tag in SELF_CLOSING_TAGS
 
         # setup node
         node_class = self.get_node_class(tag, node_kwargs)
         node = node_class(**node_kwargs)
 
+        for name, value in attrs:
+            if value is None:
+                value = ''
+
+            if name == 'id':
+                node.id_list.extend(self.parse_attribute_list(value))
```

Comment:
nit: Seems this should work as well
```suggestion
                node.id_list.extend(value.split())
```

Reply:
@maratori Ouh! I didn't know that ``str.split()`` can remove empty strings